packages/rancher-windows-gmsa/charts/templates/deployment.yaml

```
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name}}
+  namespace: {{.Release.Namespace}}
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      serviceAccountName: {{ .Release.Name }}
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/os
+                    operator: In
+                    values:
+                      - linux
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ .Release.Name }}
+          image: '{{ template "system_default_registry" . }}{{ .Values.image.repository }}:{{ .Values.image.tag }}'
+          imagePullPolicy: {{ .Values.image.imagePullPolicy }}
+          readinessProbe:
+            httpGet:
+              scheme: HTTPS
+              path: /health
+              port: 443
+          ports:
+            - containerPort: 443
+          volumeMounts:
+            - name: tls
+              mountPath: "/tls"
+              readOnly: true
+          env:
+            - name: TLS_KEY
+              value: /tls/key
+            - name: TLS_CRT
+              value: /tls/crt
+      volumes:
+        - name: tls
+          secret:
+            secretName: {{ .Values.certificates.secretName }}
+            items:
+              - key: tls.key
+                path: key
+              - key: tls.crt
+                path: crt

```

nit: I prefer keeping file extensions if possible. What do you think?

```suggestion
            - name: TLS_KEY
              value: /etc/ssl/rancher-windows-gmsa-webhook/tls.key
            - name: TLS_CRT
              value: /etc/ssl/rancher-windows-gmsa-webhook/tls.cert
      volumes:
        - name: /etc/ssl/rancher-windows-gmsa-webhook
          secret:
            secretName: {{ .Values.certificates.secretName }}
            items:
              - key: tls.key
                path: tls.key
              - key: tls.crt
                path: tls.cert
```